modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -5,13 +5,14 @@ You can configure an external PostgreSQL instance by using the Helm Chart. By de
 
 .Prerequisites
 
+* You are using PostgreSQL version 15.
```

Comment:
Per [RHIDP-2016](https://issues.redhat.com/browse/RHIDP-2016).  Postgres 14, 15, 16 are supported.  This will be detailed on the Developer Hub life cycle page.  This statement should be amended to:

> You are using a supported version of PostgreSQL

It should also be the same on the Operator guidance.